osCam/core/views.py

```diff
@@ -119,14 +130,21 @@ def rescaleFrame(frame, scale):
 
 @login_required
 def home(request):
-    return render(request, 'core/home.html')
+	
+    return render(request, 'core/home.html', {
```

Comment:
assign the context dict to a variable and pass that to render for clarity rather than putting the whole thing in the argument.

Reply:
fixed